DirectApiRequestHasMoreThanOneAuthorization Error

Whenever I try to post to an Azure Logic App, I get this message

> ##[error]{"message":{"error":{"code":"DirectApiRequestHasMoreThanOneAuthorization","message":"The request has both SAS authentication scheme and 'Basic' authorization scheme. Only one scheme should be used."}}}

I can confirm I can post to this url from PostMan with no issue, here is my input

```
    url: ***
    method: POST
    contentType: application/json
    data: { "url": "http://isaaclevin.com/about" }
    timeout: 5000
```

With logic apps, the auth is in the url, so I do not supply a Bearer or Basic Auth. I have a feeling that since I am not supplying either, the http request looks odd.

--- a/src/index.js
+++ b/src/index.js
@@ -1,7 +1,7 @@
 const core = require("@actions/core");
 const axios = require("axios");
 
-const auth = {}
+let auth = undefined
 let customHeaders = {}
 
 if (!!core.getInput('customHeaders')) {
@@ -14,12 +14,11 @@ if (!!core.getInput('customHeaders')) {
 
 const headers = { 'Content-Type': core.getInput('contentType') || 'application/json' }
 
-if (!!core.getInput('username')) {
-  auth.username = core.getInput('username');
-}
-
-if (!!core.getInput('password')) {
-  auth.password = core.getInput('password');
+if (!!core.getInput('username') || !!core.getInput('password')) {
+  auth = {
+    username: core.getInput('username'),
+    password: core.getInput('password')
+  }
 }
 
 if (!!core.getInput('bearerToken')) {
@@ -31,7 +30,6 @@ const instance = axios.create({
   timeout: parseInt(core.getInput('timeout') || 5000, 10),
   headers: { ...headers, ...customHeaders }
 });
-
 
 (async() => {
   try {

